models/comment-models.js

```diff
@@ -0,0 +1,29 @@
+const db = require("./../db/connection.js");
+const format = require("pg-format");
+
+exports.selectComments = (reviewId) => {
+  let queryStr = 'SELECT * FROM comments'
+  let queryParam = []
+
+
+  const regEx = /^\d+$/gm;
+
+  if (regEx.test(parseInt(reviewId)) === false) {
```

Comment:
If it is an invalid ID (such as a string) PSQL should automatically throw and error due to it breaking the sql type - you can then instead handle this error in your error handler by checking the psql error code :) We won't need to deal with it here, only the error from an id that does not exists such as 10000) - then we can do a promise .reject if we get no data back